src/app/home/home.component.html

```diff
@@ -2,5 +2,6 @@
 <sw-countdown></sw-countdown>
 <sw-about swScrollSpy id="about"></sw-about>
 <sw-registration-wizard swScrollSpy id="registration"></sw-registration-wizard>
+<sw-mentors swScrollSpy id="mentors"></sw-mentors>
```

Comment:
Place this above the wizard. We need to first show everything from the main event, then show the mentors